### PR TITLE
Re-enable GraphSON testing for gremlin-python

### PR DESCRIPTION
**Pull request by xiazcy**
Updated to handling aggregating results as GraphSON doesn't deserialize streams, re-enabled GraphSON testing in tests, and updated GraphSON class naming to be more consistent.

### CODE REVIEW

File: gremlin-python/src/main/python/tests/conftest.py [hunk 1]
```diff
@@ -110,11 +111,17 @@ def graphbinary_serializer_v4(request):
...
 
-@pytest.fixture(params=['graphbinaryv4'])
+@pytest.fixture(params=['graphbinaryv4','graphsonv4'])
 def remote_connection(request):
     try:
         if request.param == 'graphbinaryv4':
-            remote_conn = DriverRemoteConnection(anonymous_url, 'gmodern')
+            remote_conn = DriverRemoteConnection(anonymous_url, 'gmodern',
+                                                 request_serializer=serializer.GraphBinarySerializersV4(),
+                                                 response_serializer=serializer.GraphBinarySerializersV4()                                                 )
```
Comment (kenhuuu) [inline gremlin-python/src/main/python/tests/conftest.py:120]: Nit: the final ) needs to be shifted left.

**Comment by kenhuuu** [comment]
VOTE +1

**Comment by Cole-Greer** [comment]
VOTE +1

**Comment by xiazcy** [comment]
VOTE +1

**Pull request merged**